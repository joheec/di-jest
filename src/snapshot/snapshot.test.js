test('All calls and the name of the mock is written as a snapshot', () => {
  // expect(mockFunc).toMatchSnapshot();
});

test('mock name', () => {
  // expect(mockFunc.mock.getMockName()).toBe('a mock name');
});
