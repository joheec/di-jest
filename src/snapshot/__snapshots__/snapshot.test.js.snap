// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main snapshot tools generated serialized value for React tree 1`] = `
<div>
  Portland, ME
</div>
`;

exports[`property matchers generated fields will fail every time 1`] = `
Object {
  "createdAt": 2019-01-28T19:33:58.109Z,
  "id": 6,
  "name": "DiJest",
}
`;

exports[`property matchers generated fields will pass every time 1`] = `
Object {
  "createdAt": Any<Date>,
  "id": Any<Number>,
  "name": "DiJest",
}
`;
